enzy_htp/_interface/modeller_interface.py

```diff
@@ -1,27 +1,179 @@
-from typing import List, Tuple
+#TODO(CJ): documentation
+import os
+from typing import List
+
 from pathlib import Path
 
 from .base_interface import BaseInterface
 
-from enzy_htp import _LOGGER
+from enzy_htp import config
+
 from enzy_htp.core import file_system as fs
-from enzy_htp._config.modeller_config import ModellerConfig, default_modeller_config
 
+from enzy_htp._config.modeller_config import ModellerConfig, default_modeller_config
+try:
+    from modeller import *
+    from modeller.automodel import *
```

Comment:
Probably make it non-wildcard import?

Reply:
Updated to a new version where the imports are in the class themselves and accessed as `@property` decorators.